aws-ts-vpc-with-ecs-fargate-py/ecs-fargate-python/__main__.py

```diff
@@ -0,0 +1,174 @@
+"""An AWS fargate with crosswalk vpc"""
+
+import pulumi
+from pulumi import export, ResourceOptions, Config, StackReference, get_stack, get_project
+import json
+import pulumi_aws as aws
+
+#https://www.pulumi.com/docs/intro/concepts/organizing-stacks-projects/#inter-stack-dependencies
+
+# read local config settings - network
+config = Config()
+
+# reading in vpc StackReference Path from local config
+mystackpath = config.get("mystackpath")
+
+# setting the StackReference
+mycrosswalkvpc = StackReference(f"{mystackpath}")
+
+# Get all network values from previously created vpc #
+pulumi_vpc = mycrosswalkvpc.get_output("pulumi_vpc_id")
+pulumi_vpc_name = mycrosswalkvpc.get_output("pulumi_vpc_name")
+pulumi_vpc_cidr = mycrosswalkvpc.get_output("pulumi_vpc_cidr")
+pulumi_vpc_id = mycrosswalkvpc.get_output("pulumi_vpc_id")
+pulumi_private_subnets = mycrosswalkvpc.get_output("pulumi_vpc_private_subnet_ids")
+pulumi_public_subnets = mycrosswalkvpc.get_output("pulumi_vpc_public_subnet_ids")
+pulumi_az_amount = mycrosswalkvpc.get_output("pulumi_vpc_az_zones")
+env_stack = get_stack()
+env_project = get_project()
+config_file = "Pulumi." + env_stack + ".yaml"
+
+# common tags
+my_tags = {"application":"fargate", "crosswalk-vpc":"yes", "demo":"yes", "costcenter":"1234", "env":"dev", "vpc_name": pulumi_vpc_name, "vpc_cidr":pulumi_vpc_cidr, "pulumi:project":env_project, "pulumi:stack":env_stack, "pulumi:Config":config_file}
```

Comment:
why add pulumi.Config here? This is a stack based thing right?

Reply:
yes, it is a stack based thing.  I went ahead and removed it.